server/test/controllers/applicant/ApplicantProgramsControllerTest.java

```diff
@@ -67,7 +68,7 @@ public void indexWithApplicantId_differentApplicant_redirectsToHome() {
   public void indexWithApplicantId_applicantWithoutProfile_redirectsToHome() {
     Result result =
         controller
-            .indexWithApplicantId(fakeRequest(), applicantWithoutProfile.id)
+            .indexWithApplicantId(fakeRequest(), applicantWithoutProfile.id, List.of())
```

Comment:
I added a test that passes in values, but am essentially only checking that the controller can still successfully redirect.  I considered how I might check that the correct programs are shown under the "Recommended" section, but at that point I'd be testing the view rather than actually unit testing controller logic.  Does that seem reasonable to you?